src/test/scala/info/bethard/timenorm/formal/TypesTest.scala

```diff
@@ -997,16 +997,29 @@ class TypesTest extends FunSuite with TypesSuite {
     assert(mar31.following(startOf1980).next === SimpleInterval.of(1980, 3, 31))
```

Comment:
It seems like none of these changes are relevant to the pull request. Though see my comments below anyway, since it looks like you have some issues with the current state of this file in your branch.